addons/website_sale/static/src/js/components/media_dialog/website_sale_media_dialog.js

```diff
@@ -0,0 +1,38 @@
+import { MediaDialog, TABS } from "@web_editor/components/media_dialog/media_dialog";
+
+export class WebsiteSaleMediaDialog extends MediaDialog {
+    
+    setup() {
+        super.setup();
+        this.addTab(TABS.VIDEOS);
+    }
+    
+    /**
+     * @override
+     */
+
+    async save() {
+        if (this.errorMessages[this.state.activeTab]) {
+            this.notificationService.add(this.errorMessages[this.state.activeTab], {
+                type: 'danger',
+            });
+            return;
+        }   
```

Comment:
```suggestion
        if (this.errorMessages[this.state?.activeTab]) {
            this.notificationService.add(this.errorMessages[this.state.activeTab], { type: 'danger' });
            return;
        }   
```

Is the error message translatable?

Reply:
Yes it is translatable. The errorMessage is prepared in a different file with _t applied on them. (Managed by MediaDialog itself)